Dockerfile

```
@@ -0,0 +1,22 @@
+
+WORKDIR /src
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN CGO_ENABLED=0 go build -o /out/engram ./cmd/engram
+
+RUN wget -O /out/us-west-2-bundle.pem https://truststore.pki.rds.amazonaws.com/us-west-2/us-west-2-bundle.pem
+
+FROM alpine:3.21
+RUN apk add --no-cache ca-certificates tzdata
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)